src/app/utils.js

```diff
@@ -252,3 +253,70 @@ export const userIsControllerAdmin = (conn) => {
   const controllerAccess = conn?.info?.user?.controllerAccess;
   return ["superuser", "admin"].includes(controllerAccess);
 };
+
+/**
+  Returns the modelStatusData filtered by the supplied values.
+  @param {Object} modelStatusData The model status data to filter
+  @param {String} appName The name of the application to filter the data by.
+*/
+export const filterModelStatusData = (modelStatusData, appName) => {
```

Comment:
Is this how we want to do this? I feel like a better approach might be to have selectors like the following so that we can use them elsewhere.
- `getUnitsByApplication()`
- `getMachinesByApplication()` 
- etc...

Reply:
Perhaps.. that's for another PR though.

Reply:
Yup, no eyeballs on it :D